src/main/java/com/example/springpractise/controller/SensitiveWordController.java

```diff
@@ -0,0 +1,58 @@
+package com.example.springpractise.controller;
+
+import com.example.springpractise.service.SensitiveWordService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.EntityNotFoundException;
+
+@RestController
+@RequestMapping("sensitiveWords")
+public class SensitiveWordController {
+    private final SensitiveWordService sensitiveWordService;
+    public SensitiveWordController(SensitiveWordService sensitiveWordService) {
+        this.sensitiveWordService = sensitiveWordService;
+    }
+
+    @GetMapping()
+    public ResponseEntity<?> getSensitiveWords() {
+        try {
+            return new ResponseEntity<>(sensitiveWordService.getSensitiveWords(), HttpStatus.FOUND);
+        } catch (
+                EntityNotFoundException E) {
+            return new ResponseEntity<>("No words found",HttpStatus.NOT_FOUND);
+        }
```

Comment:
you can refactor how your response is sent such that you only have one method to structure your responses like we discussed yesterday. That way your code becomes easier to update and manage